dts/arm/st/g0/stm32g0_crypt.dtsi

```diff
@@ -10,6 +10,15 @@
 	};
 
 	soc {
+		aes: aes@40026000 {
```

Comment:
Would be nice to enable on a board, so it could be at least built in CI

Reply:
The only official ST boards with HW AES support are STM32G081B-EVAL and STM32G0C1E-EV.
I assume neither are widely distributed.
But regular CI builds and eventually test runs would indeed be very helpful.
Would it be fine if we add such a board later in a extra pr? Would anybody of you have access to such a board to run tests, or would it be for build tests only?